lib/models/types/moment.js

```diff
@@ -35,7 +31,7 @@ class SchemaTypeMoment extends SchemaType {
   }
 
   match(value, query, data) {
-    return value ? value.valueOf() === query.valueOf() : false;
+    return !!value && value.valueOf() === query.valueOf();
```

Comment:
```suggestion
    return Boolean(value) && value.valueOf() === query.valueOf();
```

Personally I prefer using `Boolean()` approach. But since it is not included in `eslint-config-hexo`, use `!!` will be fine as well.